web/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.3.0",
   "private": true,
   "scripts": {
-    "dev": "cross-env NODE_OPTIONS=\"-r next-logger/presets/next-only\" next dev -p 3000 | pino-pretty",
+    "dev": "cross-env NODE_OPTIONS=\"-r next-logger/presets/next-only --inspect\" next dev -p 4000 | pino-pretty",
```

Comment:
Can we update the readme to reflect the port number please?

Reply:
Updated the next-web readme